Source/Core/Core/PowerPC/Interpreter/Interpreter_SystemRegisters.cpp

```diff
@@ -554,7 +554,7 @@ void Interpreter::mffsx(UGeckoInstruction inst)
   // TODO(ector): grab all overflow flags etc and set them in FPSCR
 
   UpdateFPSCR();
-  riPS0(inst.FD) = 0xFFF8000000000000 | FPSCR.Hex;
+  rPS(inst.FD).SetPS0(UINT64_C(0xFFF8000000000000) | FPSCR.Hex);
```

Comment:
curious how you decide to use UINT64_C or ULL

Reply:
`UINT64_C` is necessary because std::uint64_t on UNIX can be defined as either `unsigned long` or `unsigned long long` (with their relevant literal specifiers being UL or ULL).

Given we have both double and u64 overloads, using a fixed literal specifier can cause errors because of ambiguity. (More condensed [example](https://godbolt.org/z/w71o3p))

Reply:
yea i suspected that was the reason...seems like most ULL should be changed to the macro for dolphin's use case, then? (there are examples of this in other lines changed in this PR's diff). Not saying it's super important, was just checking.